src/site/content/en/lighthouse-performance/performance-scoring/index.md

```diff
@@ -120,6 +125,48 @@ The tables below show the weighting for each metric in each version of Lighthous
   </table>
 </div>
 
+
+
+### How metric scores are determined {: #metric-scores }
+
+Once Lighthouse is done gathering the raw performance metrics (mostly reported in miliseconds), it converts them into a score by mapping the raw performance number into a metric score from 0 to 100 by looking where the metric value falls on the Lighthouse scoring distribution. The scoring distribution is
+a log-normal distribution derived from the performance metrics of real website performance
+data on [HTTP Archive](https://httparchive.org/).
+
+For example, Largest Contentful Paint (LCP) measures when a user perceives that the
+largest content of a page is visible. The metric value for LCP represents the time duration between
+the user initiating the page load and the page rendering its primary content. Based on real
+website data, top-performing sites render LCP in about 1,220ms, so that metric value is mapped to
+a score of 99.
+
+Going a bit deeper, the Lighthouse scoring curve model uses HTTPArchive data to determine two control points that then set the shape of a [log-normal](https://en.wikipedia.org/wiki/Weber%E2%80%93Fechner_law) curve. The 25th percentile of HA data becomes a score of 50 (our median control point). And the 5th percentile of HA data becomes a score of 95-ish (our PODR - point of diminishing returns control point).
+
+
+
+<figure class="w-figure">
+  <img src="./scoring-curve.png" alt="Image of a the scoring curve for TTI" style="max-width: 600px;">
+  <figcaption class="w-figcaption">
+    <a href="https://www.desmos.com/calculator/dufar5rf4g">Explore the scoring curve for TTI</a>.
+  </figcaption>
+</figure>
+
+
+
+### How scores are color-coded {: #color-coding }
+
+The metrics scores and the perf score are colored according to these ranges:
+
+* 0 to 49 (slow): Red
+* 50 to 89 (moderate): Orange
+* 90 to 100 (fast): Green
+
+### What can developers do to improve their performance score?
+First, use the [Lighthouse scoring calculator](https://paulirish.github.io/lh-scorecalc/) to help understand what thresholds you should be aiming for achieving a certain Lighthouse performance score.
+
+In the Ligthhouse report, the _Opportunities_ section has detailed suggestions and documentation on how to implement them. Additionally, the _Diagnostics_ section lists additional guidance that developers can explore to further experiment and tweak with their performance.
+
+
+<!--
```

Comment:
is this meant to be commented out?

Reply:
Yes, I wanted to keep it there for historical reasons, but I don't think its important for the typical user. HTML comment aint a great solution but it's not terrible. 🤷‍♂️

Reply:
maybe add that to the comment then? :)
```suggestion
<!--
We don't think users care about the historical scoring rubrics, but we'd still prefer to keep them around because X
```

Reply:
please fill out "X" :P